src/ConnectedMode/Embedded/SonarLintTargets.xml

```diff
@@ -1,4 +1,59 @@
 ﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup>
+<!-- TODO 
+
+* decide the final root directory location
+* remove the debug Target
+* decide whether to include $(DesignTimeBuild) in the Conditions.
+
+-->
+
+  <!-- 
+      This targets file was added by SonarLint for Visual Studio.
+      If you are no longer using SonarLint for Visual Studio then you can safely delete it.
+      
+      It is only active for builds taking place inside Visual Studio, and does not impact the content of the assembly being built.
+ 
+      Its purpose is to ensure the correct configuration for the Sonar C# and Sonar VB.NET Roslyn rules is used
+      when a solution is in Connected Mode i.e. when it is bound to a SonarQube or SonarCloud project.
+      
+  -->
+  
+  <PropertyGroup Condition="$(BuildingInsideVisualStudio) == 'true'">
+    <!-- C#/VB.NET settings need to be in different folders -->
+    <_SLVSRootFolder>$(LOCALAPPDATA)\SonarLint\VS\$(SolutionName)_$(SolutionDir.GetHashCode())\$(Language)</_SLVSRootFolder>
+    
+    <_SLVSGeneratedRoslynConfigFile>$(_SLVSRootFolder)\slvs_generated.globalconfig</_SLVSGeneratedRoslynConfigFile>
+    <_SLVSGeneratedAdditionalFile>$(_SLVSRootFolder)\SonarLint.xml</_SLVSGeneratedAdditionalFile>
+   </PropertyGroup>
+
+  <ItemGroup Condition="$(BuildingInsideVisualStudio) == 'true' AND Exists($(_SLVSGeneratedRoslynConfigFile))">
+    <EditorConfigFiles Condition="Exists($(_SLVSGeneratedRoslynConfigFile))" Include="$(_SLVSGeneratedRoslynConfigFile)" />
+    <AdditionalFiles Condition="Exists($(_SLVSGeneratedAdditionalFile))" Include="$(_SLVSGeneratedAdditionalFile)" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+
+  
+  
+  
+  
+ 
+  <Target Name="SLVS_DebugOutput_TODO_Remove" BeforeTargets="CoreCompile" >
```

Comment:
This target can be removed, leaving only the comment and the Properties/ItemGroups.